src/corvus_python/pyspark/utilities/__init__.py

```diff
@@ -1,5 +1,29 @@
 # flake8: noqa
-from .dataframe_utils.null_safe_join import null_safe_join
 from .spark_utils.spark_utils import get_spark_utils
-from .session_utils.local_spark_session import LocalSparkSessionConfig, LocalSparkSession
-from .session_utils.get_or_create_spark_session import get_or_create_spark_session
\ No newline at end of file
+
+# Only import pyspark-dependent modules if pyspark is available
+from .._optional_imports import check_pyspark_available
```

Comment:
Where does this module live?

Reply:
This is a leftover from a previous refactoring attempt